sdk/webpubsub/azure-messaging-webpubsubservice/azure/messaging/webpubsubservice/_operations/_operations.py

```diff
@@ -106,14 +109,14 @@ def build_close_all_connections_request(
 
 
 def build_send_to_all_request(
-    hub: str, *, json: Any = None, content: Any = None, excluded: Optional[List[str]] = None, **kwargs: Any
+    hub: str, *, content: IO, excluded: Optional[List[str]] = None, filter_string: Optional[str] = None, **kwargs: Any
```

Comment:
should this be called `filter` instead?

Reply:
`filter` cannot pass CI check, due to name conflict with python built-in function `filter`. So I renamed it to `filter_string`.

Reply:
@xingsy97 Could you show the CI failure about `filter`? The name `filter` is widely used in many packages and we could suppress the check if necessary

Reply:
run pylint 
`python G:\azure-sdk-for-python\eng\tox\run_pylint.py -t G:\azure-sdk-for-python\sdk\webpubsub\azure-messaging-webpubsubservice`

See errors like
```
azure\messaging\webpubsubservice\aio\_operations\_patch.py:336: [W0622(redefined-builtin), WebPubSubServiceClientOperationsMixin.send_to_user] Redefining built-in 'filter'
azure\messaging\webpubsubservice\_operations\_operations.py:773: [W0622(redefined-builtin), WebPubSubSerubServiceClientOperationsMixin.send_to_all] Redefining built-in 'filter'
```